src/zope/i18n/tests/test_translationdomain.py

```diff
@@ -115,13 +115,15 @@ def testMessageIDRecursiveTranslate(self):
                           translate, msgid1, None, None, 'en', "default")
         # Recusrive translations also work if the original message id wasn't a
         # message id but a unicode with a directly passed mapping
-        self.assertEqual("Color: BLUE/YELLOW",
-                         translate(u"Color: ${color1}/${color2}", mapping=mapping,
-                                   target_language='en'))
-
-        # If we have mapping with a message id from a different domain, make sure
-        # we use that domain, not ours. If the message domain is not registered yet,
-        # we should return a defualt translation.
+        self.assertEqual(
+            "Color: BLUE/YELLOW",
+            translate(u"Color: ${color1}/${color2}", mapping=mapping,
+                      target_language='en'))
+
+        # If we have mapping with a message id from a different
+        # domain, make sure we use that domain, not ours. If the
+        # message domain is not registered yet, we should return a
+        # defualt translation.
```

Comment:
Typo: defualt -> default.

(Yeah, it's an old typo, but you touched the code last ;)

(Also, a few lines above there's a "Recusrive" typo.)